src/main/resources/stylesheets/snyk_code_suggestion.scss

```diff
@@ -27,7 +27,7 @@ a,
 }
 
 // TODO: remove ignore styling at the end of cycle 5 2025
-// TODO: keep the font-size
+// TODO: keep the font-size - it should come from Language Server
```

Comment:
Actually, shouldn't it come from the IDE settings? https://www.perplexity.ai/search/intellij-sdk-editorcolorsschem-z5REg62VRIOT53bY4zTslg#1

Reply:
I'm not sure why we need to "_remove ignore styling at the end of cycle 5 2025. Keep the font-size_", but I just wanted to note that in my opinion, if possible, move as much as we can to Language Server so we have less repeated rules spread across IDEs. 

The class `ignore-warning` is defined in Language Server and we are overriding the same rules with the same values form IntelliJ but indeed using a custom `font-size`. With this PR, properties related to `font` are relative to the `root` definition so it could be that this `ignore-warning` component does not need to have its custom font-size specified from IntelliJ if this component stays.

I'll remove my comment to not cause confusion :)

<img width="1299" alt="ide-vs-ls" src="https://github.com/user-attachments/assets/b85ffa3c-c347-48a7-afac-8eff047b8ec5">

<img width="1828" alt="ignores-warning" src="https://github.com/user-attachments/assets/2d2a3187-a83d-47ae-8f7c-21b8bd545e97">